### PR TITLE
feat: Add SecureRandom fuzz payload generator JS script

### PR DESCRIPTION
**Pull request by kingthorin**
- payloadgenerator/securerandom.js > New payload generator script which uses SecureRandom.
- CHANGELOG.md > Added note.

Related to zaproxy/zaproxy#6892

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: payloadgenerator/securerandom.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+/**
+ * Returns the next generated payload.
+ * 
+ * This method is called while hasNext() returns true.
+ * 
+ * @return {string} The next generated payload.
+ */
+function next() {
+	count++;
+	// There are other other data type options offered by SecureRandom
```
Comment (kingthorin) [inline payloadgenerator/securerandom.js:38]: Dupe “other”

File: payloadgenerator/securerandom.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+/**
+ * Returns the next generated payload.
+ * 
+ * This method is called while hasNext() returns true.
+ * 
+ * @return {string} The next generated payload.
+ */
+function next() {
+	count++;
+	// There are other data type options offered by SecureRandom
```
Comment (thc202) [inline payloadgenerator/securerandom.js:38]: I'd remove all the other comments, we already have the templates if more info is needed.

Comment (kingthorin) [reply payloadgenerator/securerandom.js:38]: You mean like the function description comments or the other lines here or both?

Comment (thc202) [reply payloadgenerator/securerandom.js:38]: The function descriptions (which are in the template).

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Comment by thc202** [comment]
If there are no other PRs ready the add-on could be released.

**Comment by kingthorin** [comment]
We could get this one in if you're good with it?
https://github.com/zaproxy/community-scripts/pull/214

**Comment by thc202** [comment]
Comment added, as long as we restore the previous behaviour and address the comment, +1.

**Pull request merged**